src/server/JasmineGraphInstanceService.cpp

```diff
@@ -1687,6 +1687,12 @@ void *instanceservicesession(void *dummyPt) {
                 instance_logger.log("Sent : " + result, "info");
             }
         } else if (line.compare(JasmineGraphInstanceProtocol::GRAPH_STREAM_START) == 0) {
+            std::string graphIdentifier = graphId + "_"+partitionId;
```

Comment:
Need space between `+`

Reply:
Fixed all the formatting issues in `src/server/JasmineGraphInstanceService.cpp` with this https://github.com/miyurud/jasminegraph/pull/132/commits/d8d887f5453408266df16065d675be6691184b72 commit

Reply:
Ok